src/jbi/whiteboard_actions/default.py

```diff
@@ -31,10 +37,111 @@ def __init__(self, **kwargs):
         self.whiteboard_tag = kwargs.get("whiteboard_tag")
         self.jira_project_key = kwargs.get("jira_project_key")
 
-    def __call__(self, payload):
+    def __call__(  # pylint: disable=too-many-locals,too-many-return-statements,inconsistent-return-statements
+        self, payload
+    ):
         """
-        Called from BZ webhook
-        Call Jira SDK with project key etc.
+        Called from BZ webhook when default action is used. All default-action webhook-events are processed here.
         """
-        print(payload)
-        print(self.parameters)
+        try:
+            payload: BugzillaWebhookRequest = BugzillaWebhookRequest.parse_obj(
+                payload
+            )  # typing assistance
+            bugzilla_client = get_bugzilla()
+            jira_client = get_jira()
+            settings = get_settings()
+
+            current_bug_info = bugzilla_client.getbug(payload.bug.id)
+            bug_obj = BugzillaBug.parse_obj(current_bug_info.__dict__)
+            target = payload.event.target
+            linked_issue_key = bug_obj.extract_from_see_also()
+
+            if target == "comment":
+                if not linked_issue_key:
+                    # noop
+                    return JSONResponse(content={"status": "noop"}, status_code=201)
+                # else
+                jira_response = jira_client.issue_add_comment(
+                    issue_key=linked_issue_key,
+                    comment=payload.map_as_jira_comment(),
+                )
+                return JSONResponse(
+                    content={"status": "comment", "jira_response": jira_response},
+                    status_code=201,
+                )
+
+            if target == "bug":
+                if linked_issue_key:
+                    # update
+                    fields, comments = payload.map_as_tuple_of_field_dict_and_comments()
+                    jira_update_response = jira_client.update_issue_field(
+                        key=linked_issue_key, fields=fields
+                    )
+                    # comment
+                    jira_comment_responses = []
+                    for comment in comments:
+                        jira_comment_responses.append(
+                            jira_client.issue_add_comment(
+                                issue_key=linked_issue_key, comment=comment
+                            )
```

Comment:
What happens here if for some reason, `jira_client.issue_add_comment` fails -- maybe there's some blip of a `500` error on Jira's end or something. Would we be in a state where only some of the comments we want on the Jira ticket appear? Should we have some sort of retry mechanism?

Reply:
> Would we be in a state where only some of the comments we want on the Jira ticket appear? 

Currently that is the case.

> Should we have some sort of retry mechanism?

I think we may want to reconsider "retries" or perhaps complete bug validations (confirm bug is 1:1 with issue--comments, values etc) whenever we receive an event.